packages/codec/lib/format/errors.ts

```diff
@@ -922,7 +923,7 @@ export interface ReadErrorStorage {
  */
 export interface StorageNotSuppliedError {
   kind: "StorageNotSuppliedError";
-  range: Storage.Range;
+  slot: BN;
```

Comment:
What?  No.  It's an error.  It gets thrown and then decoding stops.  Also this is just the code for the format, a comment like this would make more on where I changed where the error is actually thrown!  But like I said it's just thrown in a different place.  It's still going to stop decoding.

Reply:
oh! I had a different mental model.